common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5840,6 +5851,14 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(0)
 
+  val WORKER_VERIFY_CHECKSUM_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.verify.checksum.enable")
```

Comment:
```suggestion
    buildConf("celeborn.worker.checksum.verify.enabled")
```

Reply:
Renamed, thank you.

Reply:
would you rebase the code? there is conflicts.
<img width="1009" alt="image" src="https://github.com/user-attachments/assets/bed8d687-93ab-41df-b285-76da3d57b841" />

Reply:
> would you rebase the code? there is conflicts.
Rebased, thank you.